### PR TITLE
Add script to move data from redis to psql

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

**Comment by TomasTomecek** [comment]
Thinking out loud: what would be a better solution: have it as a standalone script, or have a separate migration for it and utilize alembic?

https://github.com/packit-service/packit-service/pull/502/files#diff-9311d455327ddfdbc8aca25513b84428R275

@lbarcziova @lachmanfrantisek WDYT?

**Comment by lachmanfrantisek** [comment]
If you look at the previous migration (with the trigger data migration), it should be easy to migrate data from redis in the similar way.

File: alembic/versions/c6250555a36c_.py [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+    DateTime,
+    ForeignKey,
+    Enum,
+    orm,
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session, relationship
+from sqlalchemy.types import PickleType
+
+from packit_service.service.models import Installation, CoprBuild
```
Comment (lachmanfrantisek) [inline alembic/versions/c6250555a36c_.py:32]: Please, copy also these models to be sure we are using the right version. (Some skeleton should be enough.)

**Comment by lbarcziova** [comment]
@lachmanfrantisek thanks! I will try to test it with the data as you mentioned

**Comment by lbarcziova** [comment]
recheck

File: alembic/versions/c6250555a36c_.py [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+        if account is not None:
+            account.status = status
+            session.add(account)
+            return account
+        else:
+            account = cls()
+            account.account_name = account_name
+            account.status = status
+            session.add(account)
+            return account
```
[the remark this answers is not in the document]
Comment (lbarcziova) [reply alembic/versions/c6250555a36c_.py:104]: Oh, I just copied these lines from the code in models, will change it.

[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply alembic/versions/c6250555a36c_.py]: (You can simplify them in the real code as well..)

**Comment by lachmanfrantisek** [comment]
> Also, what's the point of copying the models instead of importing them?

If you do the migration in the future, you will not have the corresponding version of the model -- we need the model that fits this state of the database.

**Comment by TomasTomecek** [comment]
> > Also, what's the point of copying the models instead of importing them?
> 
> If you do the migration in the future, you will not have the corresponding version of the model -- we need the model that fits this state of the database.

true, that's clever

**Comment by lbarcziova** [comment]
Can we merge this and try it on stage?

File: alembic/versions/c6250555a36c_.py [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+depends_on = None
+
+if TYPE_CHECKING:
+    Base = object
+else:
+    Base = declarative_base()
+
+
+# Redis models
+class Model:
```
Comment (lachmanfrantisek) [inline alembic/versions/c6250555a36c_.py:48]: Can we rename that to `RedisModel`?

(I was a bit confused about the following classes what models are they -- postgres x redis.)

Comment (lachmanfrantisek) [reply alembic/versions/c6250555a36c_.py:48]: Maybe also rename the following following classes so the upgrade code can be more readable.

**Comment by lbarcziova** [comment]
@lachmanfrantisek thanks! I will rename the classes and merge

**Pull request merged**